dt-core/release-notifications.php

```diff
@@ -25,23 +25,28 @@ function dt_release_modal() {
         $last_release_notification = '1.0.0';
     }
 
-    if ( version_compare( $last_release_notification, $show_notification_for_theme_version, '>=' ) ){
-        return;
-    }
+    // if ( version_compare( $last_release_notification, $show_notification_for_theme_version, '>=' ) ){
```

Comment:
@prykon I'll really glad you value our work so highly. But maybe our user's don't need constant reminders? :D